<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowDateTime</class>
 <widget class="QDialog" name="ShowDateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="show_Date">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ShowDate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showTime">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_Time">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="show_date_time">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>150</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="show_Date_Time">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Date Time</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
